.github/workflows/ci-instrumentation.yml

```diff
@@ -51,6 +51,7 @@ jobs:
           - restclient
           - rspec
           - sinatra
+          - <instrumentation_name>
```

Comment:
This is causing issue. <value> is not allowed in this kind workflow

Reply:
Could you also check [use-conventional-commit-messages](https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/CONTRIBUTING.md#use-conventional-commit-messages) to update the title of PR